Makefile

```diff
@@ -140,6 +145,15 @@ NGC_PYTORCH_HPC_REPO := pytorch-ngc-hpc-dev
 NGC_TF_REPO := tensorflow-ngc-dev
 NGC_TF_HPC_REPO := tensorflow-ngc-hpc-dev
 
+INFINITYHUB_PYTORCH_PREFIX := rocm/pytorch
+INFINITYHUB_TENSORFLOW_PREFIX := rocm/tensorflow
+INFINITYHUB_PYTORCH_VERSION := 2.1.2
+INFINITYHUB_TENSORFLOW_VERSION := 
+export INFINITYHUB_PYTORCH_REPO := pytorch-infinityhub-dev
+INFINITYHUB_PYTORCH_HPC_REPO := pytorch-infinityhub-hpc-dev
+INFINITYHUB_TF_REPO := tensorflow-infinityhub-dev
+INFINITYHUB_TF_HPC_REPO := tensorflow-infinityhub-hpc-dev
```

Comment:
Is TF ROCm images out of the scope of this PR?

Reply:
Removed TF references